dotnet/TranslateText/Program.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Text;
+using Amazon.Translate;
+using Amazon.Translate.Model;
+using Newtonsoft.Json;
+
+
+class RequestPayload
+{
+    public string? Provider { get; set; }
+    public string? From { get; set; }
+    public string? To { get; set; }
+    public string? Text { get; set; }
+}
+class Program
```

Comment:
The function structure is incorrect and should be as shown in the [.NET Example here in our docs](https://appwrite.io/docs/functions#writingYourOwnFunction)